frontend/src/libs/components/pagination/pagination.tsx

```diff
@@ -0,0 +1,103 @@
+import { getValidClassNames } from '~/libs/helpers/helpers.js';
+import { useCallback, useState } from '~/libs/hooks/hooks.js';
+
+import { Button } from '../button/button.jsx';
+import styles from './styles.module.scss';
+
+type Properties = {
+  pagesRange: number;
+  paginationLength?: number;
+  onClick: (n: number) => void;
+};
+
+const Pagination: React.FC<Properties> = ({
+  pagesRange,
+  paginationLength = pagesRange > 5 ? 5 : pagesRange,
+  onClick,
+}: Properties) => {
+  const [currentPage, setCurrentPage] = useState(1);
+  const middleValue = Math.floor((paginationLength - 1) / 2);
```

Comment:
Create a helper for it and avoid magic numbers

Reply:
Helper for middleValue?